### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by mathbunnyru**

### CODE REVIEW

File: docs/using/recipes.md [hunk 1]
```diff
@@ -177,17 +175,15 @@ Once built, run using the command:
...
 
-Ref:
-[https://github.com/jupyter/docker-stacks/issues/999](https://github.com/jupyter/docker-stacks/issues/999)
+Ref: <https://github.com/jupyter/docker-stacks/issues/999>
 
 ## Let's Encrypt a Notebook server
 
 See the README for the simple automation here
-[https://github.com/jupyter/docker-stacks/tree/master/examples/make-deploy](https://github.com/jupyter/docker-stacks/tree/master/examples/make-deploy)
+<https://github.com/jupyter/docker-stacks/tree/master/examples/make-deploy>
```
Comment (consideRatio) [inline docs/using/recipes.md:183]: Is this correct? I'm not sure how this is parsed, I've never used `<>` associated with links until the markdown formatted introduced in this repo started making them show up.

**Review by consideRatio — APPROVED** [review]
LGTM!

File: docs/using/recipes.md (hunk 1 above)
Comment (consideRatio) [reply docs/using/recipes.md:183]: Nevermind i'm quite confident it is. I assume `<>` means "turn this into a link, where the label is the target.

Comment (mathbunnyru) [reply docs/using/recipes.md:183]: Yes, these links are official and work well.
Just a simple syntax for a simple case:
https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses

**Pull request merged**